src/schema/v2/edition_set.ts

```diff
@@ -35,7 +37,27 @@ export const EditionSetType = new GraphQLObjectType<any, ResolverContext>({
 
     availability: {
       type: GraphQLString,
-      resolve: ({ artwork }) => artwork.availability,
+      resolve: ({ availability }) => availability,
```

Comment:
@mzikherman - Fixed here

Reply:
Merging to get this in, as this _may_ be a bug if something on the front-end is displaying edition set availability using this field. Looking at Force and i am _not_ finding any fragment on edition set that references this field 🤞